scadalts-ui/src/views/EventList.vue

```diff
@@ -76,40 +254,173 @@
 					</v-form>
 				</v-col>
 			</v-row>
+			<v-row class="pb-2">
+				<v-col cols="6" v-show="selectedEvents.length">
+					<v-btn small @click="acknowledgeSelectedEvents" class="mr-2" color="blue">
+						<v-icon class="mr-2" >
+							mdi-checkbox-marked-circle-outline
+						</v-icon>
+						{{$t("eventList.acknowledgeSelectedEvents")}}</v-btn>
+					<v-btn small @click="silenceSelectedEvents" title="silence"  color="blue" class="mr-2" >
+						<v-icon class="mr-2" >
+							mdi-volume-mute
+						</v-icon>
+						{{$t("eventList.silenceSelected")}}
+						</v-btn>
+					<v-btn small @click="unsilenceSelectedEvents"  color="blue">
+						<v-icon class="mr-2" >
+							mdi-volume-high
+						</v-icon>
+						{{$t("eventList.unsilenceSelected")}}</v-btn>
+				</v-col>
+				<v-col :cols="selectedEvents.length ? 6 : 12" class="text-right">
+					<v-btn small @click="askForAckAll" class="mr-2" color="red">
+						<v-icon class="mr-2" >
+							mdi-checkbox-marked-circle-outline
+						</v-icon>
+						{{$t("eventList.acknownledgeAll")}}</v-btn>
+					<v-btn small @click="askForSilenceAll" color="red">
+						<v-icon class="mr-2" >
+							mdi-volume-mute
+						</v-icon>
+						{{$t("eventList.silenceAll")}}</v-btn>
+				</v-col>
 
+
+    		</v-row>
 			<v-data-table
+				id='eventList'
+				v-model="selectedEvents"
+				show-select
 				:headers="headers"
 				:items="eventList"
 				:options.sync="options"
 				:loading="loading"
 				:server-items-length="totalEvents"
+				:footer-props="{
+					'items-per-page-options': [10, 20, 50, 100]
+				}"
 				multi-sort
 				class="elevation-1"
 				@click:row="open"
-				></v-data-table>
-				
-		</v-container>
-		<v-progress-circular v-else indeterminate color="primary"></v-progress-circular>
-		<pre>{{searchFilters}} {{eventList}}</pre>
-	</div>
+				>
+					<template v-slot:item.activeTs="{ item }">
+						{{ $date(item.activeTs).format('YYYY-MM-DD hh:mm:ss') }}
+					</template>
+					
+					<template v-slot:item.ackTs="{ item }">
+						<span v-if="item.ackTs">{{$date(item.ackTs).format('YYYY-MM-DD hh:mm:ss') }}</span>
+						<v-icon v-else
+							class="mr-2"
+							title="acknowledge"
+							border=0
+							@click.stop="acknowledgeEvent(item);return false"
+						>
+							mdi-checkbox-marked-circle
+						</v-icon>
+					</template>
+					<template v-slot:item.alarmLevel="{ item }">
+						<img :src="(!!item.rtnTs ? alarmFlags : alarmFlagsOff)[item.alarmLevel].image">
+					</template>
+					<template v-slot:item.typeId="{ item }">
+						{{ $t(`eventList.sourceType${item.typeId}`) }}
+					</template>
+					<template v-slot:item.message="{ item }">
+						{{ item.message.split('|').filter(x => x).join(', ') | truncate }}						
+					</template>
```

Comment:
The i18n message translations are still missing. It does not look good if user see message: *"event.detector.changeCount, bin,..."*, or *"event.login, admin"*. In "en.json" file there are "event.system.userLogin" and etc. message keys defined for each message category but the translations for specific event messages are missing. For data point events there are "event.detector.lowLimit" and etc. keys already defined. Just use them in the way how it was done in PointDetails page.